### PR TITLE
Feature/separate header and footer

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: javascript-practice/src/footer.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		</nav>
+		<div class="footer-register">
+			<p class="title-register">Join our mailing list</p>
+			<div class="form-register">
+				<input class="form-input" type="text" placeholder="your@email.com" />
+				<button class="btn secondary-btn form-button">Sign up</button>
+			</div>
+		</div>
+	</div>
+	<div class="footer-line"></div>
```
Comment (phuongbuimai-agilityio) [inline javascript-practice/src/footer.html:47]: Do you need make a html element just style for a line?

File: javascript-practice/src/footer.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+					<li class="item-link">Vacancies</li>
+					<li class="item-link">Contact us</li>
+					<li class="item-link">Privacy</li>
+					<li class="item-link">Returns policy</li>
+				</ul>
+			</div>
+		</nav>
+		<div class="footer-register">
+			<p class="title-register">Join our mailing list</p>
+			<div class="form-register">
```
Comment (phuongbuimai-agilityio) [inline javascript-practice/src/footer.html:41]: Do we have form tag?

File: javascript-practice/src/footer.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<div class="container">
```
Comment (phuongbuimai-agilityio) [inline javascript-practice/src/footer.html:1]: Use too many div tag. Please update and use the semantic HTML tag

File: javascript-practice/src/header.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	<div class="header-search">
+		<img class="search-icon" src="assets/icons/search.svg" alt="icon search" />
+		<img class="close-icon" src="assets/icons/close.svg" alt="icon close" />
+	</div>
+	<div class="search-box">
+		<input class="search-input" type="text" placeholder="Search here ">
+	</div>
+	<h1 class="nav-text">Avion</h1>
+	<div class="nav-menu">
+		<h1 class="menu-text">Welcome to furniture website</h1>
```
Comment (phuongbuimai-agilityio) [inline javascript-practice/src/header.html:11]: In a page have 2 h1 tags?

File: javascript-practice/src/header.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+		<img class="menu-icon" src="assets/icons/menu.svg" alt="icon menu" />
+		<div class="menu-toggle">
+			<img class="cart-icon" src="assets/icons/shopping-cart.svg" alt="icon cart" />
+			<img class="avatar-icon" src="assets/icons/user-avatar.svg" alt="icon avatar" />
+		</div>
+	</div>
+</div>
+<div class="header-line"></div>
```
Comment (phuongbuimai-agilityio) [inline javascript-practice/src/header.html:19]: The same with footer line

File: javascript-practice/src/header.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+	<div class="nav-menu">
+		<h2 class="menu-text">Welcome to furniture website</h2>
+		<img class="menu-icon" src="assets/icons/menu.svg" alt="icon menu" />
+		<div class="menu-toggle">
+			<img class="cart-icon" src="assets/icons/shopping-cart.svg" alt="icon cart" />
+			<img class="avatar-icon" src="assets/icons/user-avatar.svg" alt="icon avatar" />
+		</div>
+	</div>
+</div>
+<hr>
```
Comment (phuongbuimai-agilityio) [inline javascript-practice/src/header.html:19]: Why do you need hr tag here?

Comment (mthanh2209) [reply javascript-practice/src/header.html:19]: Because in the design, it has 1 line

**Comment by hungnguyenhuu-agilityio** [comment]
Conflict and the comments are hangs on. Pls help check again

File: javascript-practice/src/styles/sections/shopping-cart.css [hunk 7]
```diff
@@ -3,13 +3,7 @@
...
 }
 
-.cart-line {
-	margin-bottom: 1rem;
-	height: 1px;
-	background: #EBE8F4;
-}
-
-.thead{
+.thead {
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/styles/sections/shopping-cart.css:6]: What does it mean?

Comment (mthanh2209) [reply javascript-practice/src/styles/sections/shopping-cart.css:6]: It means table header. I've fixed it to be more clear

**Pull request merged**